xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-authorization/xwiki-platform-security-authorization-api/src/main/java/org/xwiki/security/authorization/cache/internal/DefaultSecurityCache.java

```diff
@@ -117,7 +124,9 @@ private Cache<SecurityCacheEntry> newCache() throws InitializationException
         CacheConfiguration cacheConfig = new CacheConfiguration();
         cacheConfig.setConfigurationId("platform.security.authorization.cache");
         LRUEvictionConfiguration lru = new LRUEvictionConfiguration();
-        lru.setMaxEntries(DEFAULT_CAPACITY);
+        int cacheCapacity = configuration.getProperty("security.authorization.cache.capacity", DEFAULT_CAPACITY);
+        logger.debug("creating security entries cache with capacity of [{}]", cacheCapacity);
+        lru.setMaxEntries(cacheCapacity);
```

Comment:
So I might be wrong but AFAIR @tmortagne told me that those infinispan caches are configurable from an XML with their configuration ID. So I don't think such thing is actually needed.

Reply:
Yes, we usually configure most caches through the Infinispan configuration file (except very old caches like the document cache, which existed long before Infinispan). See https://extensions.xwiki.org/xwiki/bin/view/Extension/Cache%20Module#HInfinispan.